docs/conf.py

```diff
@@ -43,16 +45,23 @@
 
 # General information about the project.
 project = u'intbitset'
-copyright = u'2013, 2014 CERN'
+copyright = u'2013, 2014, 2015 CERN'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '2.1.1'
+
+# Get the version string. Cannot be done with import!
+with open(os.path.join('..', 'flask_menu', 'version.py'), 'rt') as f:
```

Comment:
Ouch! Not spotted! Thanks. Do you actually have some example of plugging document creation into Travis to test it?